_overviews/cheatsheets/index.md

```diff
@@ -26,270 +26,267 @@ permalink: /cheatsheets/index.html
       <td> </td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>var x = 5</code></pre><br /> <span class="label success">Good</span><br><pre class="highlight"><code>x=6</code></pre></td>
-      <td>variable</td>
+      <td><pre class="highlight"><code>var x = 5</code></pre><br /> <span class="label success">Good</span><br><pre class="highlight"><code>x = 6</code></pre></td>
+      <td>Variable.</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>val x = 5</code></pre><br /> <span class="label important">Bad</span><br><pre class="highlight"><code>x=6</code></pre></td>
-      <td>constant</td>
+      <td><pre class="highlight"><code>val x = 5</code></pre><br /> <span class="label important">Bad</span><br><pre class="highlight"><code>x = 6</code></pre></td>
+      <td>Constant.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>var x: Double = 5</code></pre></td>
-      <td>explicit type</td>
+      <td>Explicit type.</td>
     </tr>
     <tr>
       <td><span id="functions" class="h2">functions</span></td>
       <td> </td>
     </tr>
     <tr>
       <td><span class="label success">Good</span><br> <pre class="highlight"><code>def f(x: Int) = { x * x }</code></pre><br /> <span class="label important">Bad</span><br> <pre class="highlight"><code>def f(x: Int)   { x * x }</code></pre></td>
-      <td>define function <br /> hidden error: without = it’s a Unit-returning procedure; causes havoc</td>
+      <td>Define function.<br />Hidden error: without <code>=</code> it’s a procedure returning <code>Unit</code>; causes havoc.</td>
```

Comment:
Side note: procedure syntax has been deprecated in 2.13.0.

Reply:
I've added a note